docs/tutorials/modify_loss.md

```diff
@@ -0,0 +1,105 @@
+# tutorial 5: Modify Loss
+
+MMDetection provides users with different loss functions. But the default configuration may be not applicable for different datasets or models, so users may want to modify a specific loss to adapt the new situation.
```

Comment:
`But the default configuration may be not applicable for different datasets or models, so users may want to modify a specific loss to adapt the new situation` -> `Users may want to try different losses rather than default configuration in some scenarios.`